mod/install.php

```diff
@@ -1,271 +0,0 @@
-<?php
-/**
- * @file mod/install.php
- */
-
-use Friendica\App;
-use Friendica\Core\Install;
-use Friendica\Core\L10n;
-use Friendica\Core\System;
-use Friendica\Database\DBA;
-use Friendica\Database\DBStructure;
-use Friendica\Util\Temporal;
-
-$install_wizard_pass = 1;
-
-function install_init(App $a) {
-
-	// $baseurl/install/testrwrite to test if rewite in .htaccess is working
-	if ($a->argc == 2 && $a->argv[1] == "testrewrite") {
-		echo "ok";
-		killme();
-	}
-
-	// We overwrite current theme css, because during install we could not have a working mod_rewrite
-	// so we could not have a css at all. Here we set a static css file for the install procedure pages
-
-	$a->setConfigValue('system', 'value', '../install');
```

Comment:
@MrPetovan responding to https://github.com/friendica/friendica/pull/6041/files#r228921398

I copied it from here

Reply:
It doesn't look like it's used anywhere in the install, I didn't check the rest of the code but I would assume it's going to be the same.

Reply:
I'll check it